src/components/Icon/Icon.tsx

```diff
@@ -6,7 +6,7 @@ import iconNames from '@naturacosmeticos/natds-icons/dist/natds-icons.json';
 import { Theme } from '../../common/themeSelectors';
 
 export type IconColors = 'default' | 'primary'
-export type IconSizes = 'small' | 'medium'
+export type IconSizes = 'micro' | 'tiny' | 'small' | 'standard' | 'medium' | 'large' | 'largex' | 'largexx' | 'huge' | 'hugex' | 'hugexx'
```

Comment:
os iconsSizes deveriam ser os mesmos dos tokens de size?
poderiamos reaproveitar o tipo do token?

ficaria algo assim:

```
export type IconSizes = keyof ISizes
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#keyof-and-lookup-types